_topic_map.yml

```diff
@@ -771,6 +771,9 @@ Topics:
 - Name: Scanning pods for vulnerabilities
   File: pod-vulnerability-scan
   Distros: openshift-enterprise,openshift-origin
+- Name: Implementing Network-Bound Disk Encryption (NBDE)
+  File: ztp-nbde-implementation-guide
```

Comment:
The filename still has `ztp-` but this document doesn't have any connection to ZTP.

Why is this?  Can we remove `ztp-` from all these filenames?

Reply:
Again, ztp is a prefix I use in my filenames for easy tracking in my repo. It does not affect the content of the document at all and is not seen by customers..

Reply:
Hi @StephenJamesSmith this is a valid concern. The filename is visible as the URL. For example, this file will be displayed in the URL as ztp-nbde-implementation-guide.html. This has implications for SEO and general navigation and findability. For example, I would be worried if a CNV doc was named telco-<filename>.

Is it possible to remove ztp and just use nbde if you want to be able to find your files easily?

Reply:
With the latest refactor, this doesn't seem to match a filename at all?  Is that going to result in a broken link?

Reply:
@lack Removed "ztp"
 from filenames, IDs, and topic comments.